src/flexbox/nasa.css

```diff
@@ -0,0 +1,27 @@
+body{
+    width: 850px;
+    margin: 0 auto;
+}
+#back{
+    background-image: url("../../photos/nasa_image.jpg");
+    width: 100%;
+    height: 100%;
+}
+Main {
+    margin: 0 auto;
+    height: 450px;
+    color: white;
+}
+
+section {
+    display: flex;
+    height: 324px;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+article {
+    width: 270px;
+    height: 374px;
+    background-color: rgb(224, 222, 222);
```

Comment:
add some padding! we never want text touching the edge of it's container. 
`padding: 20px;`